packages/embed/src/js/doc-verification.js

```diff
@@ -174,12 +218,12 @@ import { getHeaders, getZipSignature } from './request';
       <smileid-combobox-trigger
         ${isSingleIdType ? 'disabled' : ''}
         ${isSingleIdType ? `value="${idTypes[0].name}"` : ''}
-        label="Select Document"
+        label="${t('pages.idSelection.selectDocument')}"
         type="button"
       >
       </smileid-combobox-trigger>
 
-      <smileid-combobox-listbox empty-label="No country found">
+      <smileid-combobox-listbox empty-label="${t('pages.idSelection.noIdTypeFound')}">
```

Comment:
Is this right?  The original text was "No country found", not 'No id type found'?